### ISSUE TITLE
A11y_SQL Azure Data Studio_Database_Create Project from database_Screen reader: Screen reader is not announcing " section Headings" of "Create project from database" section. 

### ISSUE THREAD

**Issue by PadminiGariki** [dotnet/maui#16819]
### Description

[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Aug2023; #SQL Azure Data Studio; #WCAG1.3.1; #ScreenReader; #Win32; #Benchmark; #FTP; #STP; #Element:Heading;

### Environment Details:​
Application Name: Sql Azure Data studio
Version: 1.45.1
OS: Win11

### Steps to Reproduce

1.Launch the application-Azure Database Project and install extension SQL Database Projects.
2.Navigate to SQL Database Project Home Screen with heading "Projects".
3.Tab to connections select your server and connect to it.
4.Few databases will appear under your connection then right click on any database
5.Tab till "Create project from database" and hit enter.
6. Now verify that NVDA/Narrator not announcing "Section Headings" of "Create project from database" section.

### Actual Results:
Screen reader is not announcing "Section Headings" of "Create project from database" section.

**Note:**
**Similar issue observed with chrome+ NVDA.**

### Expected Results:
Screen reader should announce "Section Headings" of "Create project from database" section.

### User Impact:
Screen reader user will not be able to know "Section Headings" of "Create project from database" section. If NVDA/Narrator not announcing.

### Attachment:
[SR is not announcing the heading along with label.webm](https://github.com/dotnet/maui/assets/142479541/f755f9e6-58cd-4804-958a-f9f87f3852eb)

[sr not announcing the heading along with label NVDA.webm](https://github.com/dotnet/maui/assets/142479541/68b47ee0-3167-4cd0-916b-3e8567c203e8)

<img width="960" alt="MicrosoftTeams-image" src="https://github.com/dotnet/maui/assets/142479541/760cb452-9f19-4a45-8f63-1e758eefa27e">

### Version with bug

7.0.92

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

**Comment by PureWeen** [comment]
Wrong Repository

**Comment by PadminiGariki** [comment]
Sorry for Inconvenience caused. Adding #Closed; tag. Will file issue on Correct Repository.
